packages/eas-cli/src/commands/build/index.ts

```diff
@@ -297,6 +297,14 @@ export default class Build extends EasCommand {
       error(EAS_UNAVAILABLE_MESSAGE, { exit: 1 });
     }
 
+    const { autoIncrement } = buildCtx.buildProfile as any; // TODO-JJ remove cast to any when android autoIncrementing is added
+    if (buildCtx.workflow === Workflow.GENERIC && autoIncrement) {
+      // Once we work out the kinks, this can be relaxed to support some GENERIC workflows.
+      // One of the known issues with extending autoIncrement to GENERIC projects is
+      // multiflavor android projects.
+      throw new Error('Autoincrementing is only supported for managed apps.');
```

Comment:
```suggestion
      error('Auto-incrementing is only supported for managed apps.', { exit: 1 });
```
So the output looks better.